api-client/src/main/java/io/split/api/client/api/ApiAttributeClient.java

```diff
@@ -0,0 +1,32 @@
+package io.split.api.client.api;
+
+import io.split.api.client.AttributeClient;
+import io.split.api.dtos.Attribute;
+import io.split.api.dtos.TrafficType;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class ApiAttributeClient implements AttributeClient {
+    public List<Attribute> list(String trafficTypeId) {
+        return new ArrayList<>();
+    }
+
+    public List<Attribute> list(TrafficType trafficType) {
+        return list(trafficType.id());
+    }
+
+    public ApiAttributeClient create(Attribute attribute) throws IllegalArgumentException {
```

Comment:
@patricioe requested creating a fluent api, so this is designed to allow for fluent execution of calls. He may have only been referring to the builder pattern for objects tho - so I'll let him clarify